mkdocs.yml

```
@@ -47,6 +47,7 @@ nav:
     - examples/aspnet.md
   - Modules:
     - modules/index.md
+    - modules/azureeventhubs.md
```

```suggestion
    - modules/eventhubs.md
```
Can we please rename the file according to the module (namespace)?